bubblepicker/src/main/java/com/igalata/bubblepicker/rendering/BubblePicker.kt

```diff
@@ -22,13 +21,6 @@ class BubblePicker : GLSurfaceView {
             field = value
             renderer.backgroundColor = Color(value)
         }
-    @Deprecated(level = DeprecationLevel.WARNING,
-            message = "Use BubblePickerAdapter for the view setup instead")
-    var items: ArrayList<PickerItem>? = null
```

Comment:
How would we set the items now?

Reply:
We we use the `BubblePickerAdapter` instead of setting the items manually.